openmicroscopy/bioformats
# JPEG - Read Physical Size X and Y

This is a reworked version of an old PR - https://github.com/openmicroscopy/bioformats/pull/2036
Original thread - https://www.openmicroscopy.org/community/viewtopic.php?f=13&t=7877&p=16128#p16128

This PR adds functionality to read the Exif tags TAG_Y_RESOLUTION and TAG_X_RESOLUTION.
Originally this PR was closed as these tags were not supported by the metadata-extractor library we were using and suitable replacements could not be found.

I have since updated the library to the latest revision enabling us to access the required tags.

To test:
We currently are not using the Physical Size X and Y, this PR simply provides users the ability to set the values on the Exif tags and have them remain as part of their metadata. The main change here is the upgrade of the metadata-extractor jar. As such it is important that this has no side affects and previous functionality is unchanged.  Ensuring that all builds and repo runs remain green would be a good indictor that the upgraded jar is working as expected.
